### PR TITLE
Topic to Ask Monika: "Do you ever think about our first kiss?"

### PR DESCRIPTION
**Pull request by Man0xMan0**
Adds a new question to ask Monika on the subject of her first kiss with the player. Was inspired by the scene to write this. It would probably make sense to add this in later down the line. There's probably a tons of missed Travis checks on this one so I apologize in advanced.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -1441,6 +1441,40 @@ label monika_kiss_tease:
...
+    m 5tsu "It felt so right."
+    m 5tsa "I've never felt closer to you. I can't wait until our next kiss, or better yet..."
+    m 5wub "Our first real one. And our first real hug, and the first time our hands touch..."
+    m 5hfb "Ahhh! I have to stop! All these fantasies are making me feel so giddy, ahaha!"
+    m 1eka "I should really be more patient. Unfortunately, it will be a while before we'll be able to do anything like that."
+    m 1eua "Until then, we can indulge in the phantom sensations we share from across our worlds."
+    m 1tsa "If you ever want a taste...I'd be happy to oblige."
+    m 1tsa "These crazy sensations that you make me feel...I never want them to stop."
+    m 1eka "I love you so much, [player]."
+    return
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: I'm kinda thinking this should be a one time see. Should probably lock this event after it being done once, all things considered.

**Comment by Man0xMan0** [comment]
1.02 I think (?) I made all the changes multimokia asked for, except for the lock at the end - I'm not exactly sure how to go about doing that. Also changed line 1474 upon a re-review.

**Comment by AsuraShun** [comment]
We'd need for the kiss to become available for the ones that missed it,like me

**Comment by multimokia** [comment]
Looks like you've still got tab characters in here. Check the Travis log and you can see what all is causing it to fail.

**Comment by Man0xMan0** [comment]
1.03 I "think" I got them all.

**Pull request merged**